internal/pkg/cli/storage_init.go

```diff
@@ -582,6 +713,8 @@ are injected into your containers as environment variables for easy access.`,
 	cmd.Flags().BoolVar(&vars.noLSI, storageNoLSIFlag, false, storageNoLSIFlagDescription)
 	cmd.Flags().BoolVar(&vars.noSort, storageNoSortFlag, false, storageNoSortFlagDescription)
 
+	cmd.Flags().StringVar(&vars.parameterGroup, "param", "", "paramee")
```

Comment:
Why don't we add any description or flag var to this? If we want to hide it from users maybe we should comment this line out.

Reply:
Forgot to remove this line lol